lzy/worker/src/main/java/ai/lzy/worker/env/DockerEnvironment.java

```diff
@@ -63,11 +63,11 @@ public void install(StreamQueue.LogHandle handle) throws EnvironmentInstallation
         try {
             prepareImage(sourceImage, handle);
         } catch (InterruptedException e) {
-            handle.logErr("Pulling image {} was interrupted", sourceImage);
+            handle.logErr("Image pulling was interrupted");
```

Comment:
But why without image name?

Reply:
It was printed for user some rows before. Originally it was the internal log and image name was helpful.